helm/ngrok-operator/templates/controller-deployment.yaml

```diff
@@ -61,6 +61,10 @@ spec:
         command:
         - /manager
         args:
+        - --app-version={{ .Chart.AppVersion }}
```

Comment:
Left a note on this up above. I think we can remove this. This app version is just the default for the helm chart and it can always be changed by supplying a different image.